docs/configuration.md

```diff
@@ -0,0 +1,214 @@
+# Configuration variables
+
+Arctic has several tuning knobs under [arctic/_config.py](https://github.com/manahl/arctic/blob/master/arctic/_config.py) which affect the functionality of certain modules.
+
+Most of these variables are initialized via environment variables, which are explained in the rest of this section.
+
+
+<br><br>
+
+## VersionStore
+
+### STRICT_WRITE_HANDLER_MATCH
+
+Controls if Arctic can only match the intended handlers for the data type. If set to true it prevents falling back to pickling if the matching handler (based on the type of data) can't serialize without objects.
```

Comment:
Is there a typical error that users see when upgrading, if we could put this below and how to clean your data then it should hopefully come up in google results.

Reply:
will attach a stacktrace (also includes the error message), removing sensitive data

Reply:
Nice, I actually meant the opposite, e.g. which types of errors that users pickling data may see. But just a suggestion, ok with this.